gettsim/social_insurance.py

```diff
@@ -1,4 +1,10 @@
-OUT_COLS = ["svbeit", "rvbeit", "avbeit", "gkvbeit", "pvbeit"]
+OUT_COLS = [
+    "sozialv_beit_m",
```

Comment:
This first can go, right?

Reply:
We keep it as an internal variable and do not return it to the user. We use it in several calculations and it makes life just easier! Is this a real problem?

Reply:
Nope!